src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/xlf/GeneralConfiguredBrowseObject.xaml.cs.xlf

```diff
@@ -0,0 +1,143 @@
+      </trans-unit>
+      <trans-unit id="BoolProperty|TreatWarningsAsErrors|Description">
+        <source>Treat warnings as errors</source>
+        <target state="new">Treat warnings as errors</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="StringProperty|TreatSpecificWarningsAsErrors|Description">
+        <source>Treat specific warnings as errors</source>
+        <target state="new">Treat specific warnings as errors</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="StringProperty|OutputPath|DisplayName">
+        <source>Output Path</source>
+        <target state="new">Output Path</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="StringProperty|DocumentationFile|DisplayName">
+        <source>Documentation file</source>
+        <target state="new">Documentation file</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="EnumProperty|GenerateSerializationAssemblies|DisplayName">
+        <source>Generate serialization assemblies</source>
+        <target state="new">Generate serialization assemblies</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="EnumValue|GenerateSerializationAssemblies.Auto|DisplayName">
+        <source>Auto</source>
+        <target state="new">Auto</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="EnumValue|GenerateSerializationAssemblies.On|DisplayName">
+        <source>On</source>
+        <target state="new">On</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="EnumValue|GenerateSerializationAssemblies.Off|DisplayName">
+        <source>Off</source>
+        <target state="new">Off</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="BoolProperty|RegisterForComInterop|DisplayName">
+        <source>Register for COM Interop</source>
+        <target state="new">Register for COM Interop</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="EnumProperty|LanguageVersion|DisplayName">
+        <source>Language version</source>
+        <target state="new">Language version</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="EnumProperty|ErrorReport|DisplayName">
+        <source>Error report</source>
+        <target state="new">Error report</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="EnumProperty|DebugInfo|DisplayName">
+        <source>Debug Info</source>
+        <target state="new">Debug Info</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="BoolProperty|CheckForOverflowUnderflow|DisplayName">
+        <source>CheckForOverflowUnderflow</source>
+        <target state="new">CheckForOverflowUnderflow</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="StringProperty|DebugSymbols|DisplayName">
+        <source>Debug symbols</source>
+        <target state="new">Debug symbols</target>
```

Comment:
This is resetting already translated strings. If we`re doing this post RTW, a hand-off/handback would recycle them. If we do it now, then we need to find a workaround, which I think would be: copy GeneralBrowseObject.*.xlf to GeneralConfigurredBrowseObject.*.xlf and rerun tool.